src/utils/date.js

```diff
@@ -27,4 +27,25 @@ const formatDateDDMMYYY = date => {
   return [pad(d.getDate()), pad(d.getMonth() + 1), d.getFullYear()].join("/");
 };
 
-export { formatToDateObject, formatDate, formatDateDDMMYYY, regexDate };
+const formatDateDDMMYYYHHMM = date => {
```

Comment:
Do we want to use https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString instead of this?